### PR TITLE
Fixes to get HTSOHM working on local machine

### PR DESCRIPTION
**Pull request by paulboone**
IMPORTANT: 
- RASPA's `simulate` must be on the PATH.

Merge guidance:
- don't accept the pull request unless you understand what ALL the code does and WHY I did it. You are responsible for your repository, so if I did something dumb, it's up to you to catch it. Don't just trust me and merge it. 
- if you have any questions or concerns, it is convention to just use github to comment directly on the line(s) of code below. 
- usually, it will be easiest to look at each commit individually, rather than all at the same time. That way you can see each change, what my commit message says it's doing, and make sure they match up.

### CODE REVIEW

File: bin/simulate.py [hunk 1]
```diff
@@ -29,8 +29,7 @@ def VoidFraction(run_ID, mat_ID):
                     "            CreateNumberOfMolecules\t0" )
     VF_input.close()
 
-    rd = os.environ['RASPA_DIR']
-    subprocess.call(shlex.split( rd + '/bin/simulate -i VoidFraction.input' ))
```
Comment (paulboone) [inline bin/simulate.py:111]: Need to check on whether RASPA version at U Netherlands also uses this syntax!

File: bin/simulate.py [hunk 2]
```diff
@@ -26,7 +26,7 @@ def VoidFraction(run_ID, mat_ID):
                     "Component 0 MoleculeName\t\thelium\n" +
                     "            MoleculeDefinition\t\tTraPPE\n" +
                     "            WidomProbability\t\t1.0\n" +
-                    "            CreateNumberOfMolecules\t0" )
+                    "            CreateNumberOfMolecules\t0\n" )
```
Comment (paulboone) [inline bin/simulate.py:107]: Need to double-check RASPA version for this as well.

**Comment by paulboone** [comment]
This PR now works with version v2.0-14-g0486f86 from Dubbeldam's branch (what Alec is currently using). His code is currently at v2.0.7-2-gb85c8a8 and our codebase doesn't currently work with that upgrade. I believe it is because the output has changed enough that we aren't able to extract the variables we need.

**Comment by akaija** [comment]
That makes sense. Would it be worthwhile to "upgrade" the codebase to
extract output in the format used in the latest version of RASPA
(v2.0.7-2-gb85c8a8)? I don't expect that to be a dificult change to
implement.

**Comment by paulboone** [comment]
It would be fine to do so, but not sure it is really worth it. Generally, I avoid upgrading unless I want a fix or a feature. You could git pull the most recent version of dubbledam’s code and run git log to see what has been committed between the version you are running and trunk. If something looks good, then upgrade! But often, it is worth waiting and only dealing with upgrades with major versions.

**Comment by paulboone** [comment]
@akaija ok so we have a reason to upgrade it now that I've branched and fixed the versioning in raspa2. I'd go ahead and make the updates to work with HEAD on https://github.com/WilmerLab/raspa2

**Pull request merged**